app/business/payments/payouts/processor/paypal/paypal_payout_processor.rb

```diff
@@ -349,6 +349,8 @@ def self.update_split_payment_state(payment)
       payment.mark_failed!
     elsif no_split_payments_are_processing
       # This means that no split payments are in the processing state. It also means that some of them have failed and some have succeeded.
+      payment.txn_id = SPLIT_PAYMENT_TXN_ID
```

Comment:
Not sure why this changes is needed, but we should not actually mark the payment as failed in this case since some parts of it have been paid out. Marking entire payment as failed will mark the entire balance as unpaid and pay out the same balance twice.